.dotnet/src/Generated/Models/AudioTranslation.Serialization.cs

```diff
@@ -26,7 +26,7 @@ void IJsonModel<AudioTranslation>.Write(Utf8JsonWriter writer, ModelReaderWriter
             writer.WritePropertyName("language"u8);
             writer.WriteStringValue(Language);
             writer.WritePropertyName("duration"u8);
-            writer.WriteNumberValue(Convert.ToInt32(Duration.Value.ToString("%s")));
+            writer.WriteNumberValue(Convert.ToDouble(Duration.Value.ToString("s\\.fff")));
```

Comment:
This change looks correct but... How did it happen? Are you using a newer version of the generator? Does this mean that the custom serialization fix for TimeSpans encoded as seconds is no longer needed?

Reply:
It must be -- that or a generator update since the customizations were added. It does appear that it's fixed, one way or another, and that the custom logic can go away!